README.adoc

```diff
@@ -58,6 +58,7 @@ The `format` method has the following options:
 * `dirtySep: String = "-DIRTY"` - will be printed before the dirty hash if the local repository is in modified state
 * `dirtyHashDigits: Int = 8` - the number of digits to be used for the dirty hash
 * `tagModifier: String => String` - allows to modify the git tag (By default this strips a leading `v` if one exists. e.g. v1.2.3 -> 1.2.3)
+* `appendSnapshot: String = "" - will append the given string at the end of the version when the current revision is not tagged (e.g. use with `"-SNAPSHOT"` to publish to Maven Snapshot repositories)
```

Comment:
Just thinking about the name. I think if we call it `appendSnapshot`, then it should only append `-SNAPSHOT`. If we want it to be more generic like it currently is, maybe we should call it something like just `append`? I actually don't know if there will be other usecases for this apart from actually using it to append `-SNAPSHOT`. Do you think changing the signature to `appendSnapshot: Boolean = false` is too specific? Then a user could just toggle it on, which is probably the most common use case for this?

Reply:
The motivation for `appendSnapshot` was, that it is only appended iff it is a _snapshot_ (a non-tagged commit in this context). Another suffix used in projects is `-DEV`, so, I'd rather keep it configurable. We could name it `appendIfSnapshot` or `appendIfUntagged`?

Reply:
or `untaggedSuffix`.

Reply:
🤔 I think either is fine in that scenario. Maybe `appendIfSnapshot` since most will be familiar with that terminology.

Reply:
ooo actually yea, I like `untaggedSuffix`.